Source/SwiftLintCore/Extensions/Configuration+LintableFiles.swift

```diff
@@ -98,7 +108,7 @@ extension Configuration {
     /// - parameter fileManager: The file manager to get child paths in a given parent location.
     ///
     /// - returns: The expanded excluded file paths.
-    private func excludedPaths(fileManager: LintableFileManager) -> [String] {
+    public func excludedPaths(fileManager: LintableFileManager) -> [String] {
```

Comment:
Perhaps add a default argument (`fileManager: LintableFileManager = FileManager.default)`) here?

Reply:
Sure. Added in a new commit